frontend/src/metabase/routes.jsx

```diff
@@ -223,6 +225,15 @@ export const getRoutes = store => (
           {getCollectionTimelineRoutes()}
         </Route>
 
+        <Route path="a/:slug" component={DataAppLanding}>
+          <ModalRoute path="move" modal={MoveCollectionModal} />
```

Comment:
do we want to allow moving data apps? doing so right now makes them duplicated, and not look like apps in the collection nav.

![Screen Shot 2022-08-23 at 10 23 30 AM](https://user-images.githubusercontent.com/30528226/186210979-0ca54b5d-c2fa-48fb-880f-ce1b44cfafd8.png)